Makefile

```diff
@@ -41,6 +41,10 @@ test: ## Run test
 test pytest tests: pipenv
 	$(PYTEST) -n4 -m 'not flaky' --dist loadfile $(flags) testsuite
 
+# Run performance tests
+performance: pipenv
```

Comment:
Sadly, this still means `make test` will run performance tests, which we do not want

Reply:
Yeah sure 🤦 sorry